src/cf-expandables.less

```diff
@@ -234,6 +234,26 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
+.expandable {
+    .expandable_cue-open {
```

Comment:
Is it necessary to nest this? It's using `.expandable` as a parent name in the BEM naming.